### ISSUE TITLE
EventReceiver logs a warning and stops receiving messages, but doesn't propagate error to client code

### ISSUE THREAD

**Issue by anthonykirby** [Azure/azure-event-hubs-java#422]
**Describe the bug**

EventHubClient logs an error without propagating it to the user, and stops receiving messages -  com.microsoft.azure.eventhubs.impl.MessagingFactory catches an unhandled exception and emits an WARN level error message. 

No exceptions are thrown to user code, but no further messages are received, and the application subsequently keeps running but idle.

**To Reproduce**

This happened three times in a couple of hours in our deployment testing environment; the application had previously been running without a problem for at least 3 weeks.  

Our code:
- creates a client `ehClient = EventHubClient.createSync(...)`
- creates a receiver `receiver = ehClient.createReceiverSync(...)`
- polls `receiver.receiveSync`

It's a guess on my part, but I suspect the issue was triggered by a brief connectivity problem or with the Event Hub that the code was connected to. 

**Expected behavior**

When an internal error like this occurs, it should be propagated to user code.  (Without propagation of the error, we end up with an "alive but dead" application.  To detect this we'd need either to scan the logs for this message - which isn't ideal - or continuously send dummy messages & restart if they're not delivered - which isn't ideal either).

**Observed behavior**

- multiple logged warnings  
`WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]`
- multiple exceptions logged  
```
WARN  c.m.a.e.impl.MessagingFactory - messagingFactory[MessagingFactory90e306], hostName[<namespace>.servicebus.windows.net], error[Unhandled exception while processing events in reactor, report this error.
java.lang.NullPointerException: uncorrelated channel: 0
...
```
- no further messages handled
- no errors propagated to client code

**Environment (please complete the following information):**
 - OS: Linux - deployed on CentOS7
 - SDK Version 
```
       <dependency>
            <groupId>com.microsoft.azure</groupId>
            <artifactId>azure-eventhubs</artifactId>
            <version>1.2.0</version>
        </dependency>
```

**Additional context**

full logs as captured by systemd (hostname & namespace name substituted for privacy).  Beforehand, the application was processing normally.  Afterwards, it was still running but useless - it responded to an  external shutdown command, but didn't receive Azure messages.

```
Jan 10 21:03:29 beta-bridge eventhub_bridge[23621]: 21:03:29.916 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:05:37 beta-bridge eventhub_bridge[23621]: 21:05:37.213 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:07:52 beta-bridge eventhub_bridge[23621]: 21:07:52.504 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:10:07 beta-bridge eventhub_bridge[23621]: 21:10:07.665 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:12:22 beta-bridge eventhub_bridge[23621]: 21:12:22.965 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:14:38 beta-bridge eventhub_bridge[23621]: 21:14:38.136 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:16:53 beta-bridge eventhub_bridge[23621]: 21:16:53.425 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:19:08 beta-bridge eventhub_bridge[23621]: 21:19:08.728 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:21:23 beta-bridge eventhub_bridge[23621]: 21:21:23.901 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:23:39 beta-bridge eventhub_bridge[23621]: 21:23:39.073 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:25:54 beta-bridge eventhub_bridge[23621]: 21:25:54.370 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:28:09 beta-bridge eventhub_bridge[23621]: 21:28:09.653 [pool-4-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: 21:32:06.400 [pool-4-thread-1] WARN  c.m.a.e.impl.MessagingFactory - messagingFactory[MessagingFactory90e306], hostName[<namespace>.servicebus.windows.net], error[Unhandled exception while processing events in reactor, report this error.
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: java.lang.NullPointerException: uncorrelated channel: 0
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.EventImpl.dispatch(EventImpl.java:112)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.dispatch(ReactorImpl.java:324)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.process(ReactorImpl.java:292)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: com.microsoft.azure.eventhubs.impl.MessagingFactory$RunReactor.run(MessagingFactory.java:490)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: java.lang.Thread.run(Thread.java:748)Cause: uncorrelated channel: 0
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleBegin(TransportImpl.java:1161)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleBegin(TransportImpl.java:67)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.amqp.transport.Begin.invoke(Begin.java:144)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleFrame(TransportImpl.java:1402)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.FrameParser.input(FrameParser.java:418)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.FrameParser.process(FrameParser.java:527)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.SaslImpl$SwitchingSaslTransportWrapper.process(SaslImpl.java:805)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.HandshakeSniffingTransportWrapper.process(HandshakeSniffingTransportWrapper.java:101)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SimpleSslTransportWrapper.unwrapInput(SimpleSslTransportWrapper.java:135)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SimpleSslTransportWrapper.process(SimpleSslTransportWrapper.java:361)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SslImpl$UnsecureClientAwareTransportWrapper.process(SslImpl.java:144)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.process(TransportImpl.java:1513)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler$1.run(IOHandler.java:233)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.SelectableImpl.readable(SelectableImpl.java:118)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler.handleQuiesced(IOHandler.java:60)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler.onUnhandled(IOHandler.java:389)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.BaseHandler.onReactorQuiesced(BaseHandler.java:87)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.BaseHandler.handle(BaseHandler.java:206)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.EventImpl.dispatch(EventImpl.java:108)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.dispatch(ReactorImpl.java:324)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.process(ReactorImpl.java:292)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: com.microsoft.azure.eventhubs.impl.MessagingFactory$RunReactor.run(MessagingFactory.java:490)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
Jan 10 21:32:06 beta-bridge eventhub_bridge[23621]: java.lang.Thread.run(Thread.java:748)]
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: 23:13:31.244 [pool-3-thread-1] WARN  c.m.a.e.impl.MessagingFactory - messagingFactory[MessagingFactory5d58d8], hostName[<namespace>.servicebus.windows.net], error[Unhandled exception while processing events in reactor, report this error.
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: java.lang.NullPointerException: uncorrelated channel: 1
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.EventImpl.dispatch(EventImpl.java:112)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.dispatch(ReactorImpl.java:324)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.process(ReactorImpl.java:292)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: com.microsoft.azure.eventhubs.impl.MessagingFactory$RunReactor.run(MessagingFactory.java:490)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: java.lang.Thread.run(Thread.java:748)Cause: uncorrelated channel: 1
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleBegin(TransportImpl.java:1161)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleBegin(TransportImpl.java:67)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.amqp.transport.Begin.invoke(Begin.java:144)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleFrame(TransportImpl.java:1402)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.FrameParser.input(FrameParser.java:418)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.FrameParser.process(FrameParser.java:527)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.SaslImpl$SwitchingSaslTransportWrapper.process(SaslImpl.java:805)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.HandshakeSniffingTransportWrapper.process(HandshakeSniffingTransportWrapper.java:101)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SimpleSslTransportWrapper.unwrapInput(SimpleSslTransportWrapper.java:135)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SimpleSslTransportWrapper.process(SimpleSslTransportWrapper.java:361)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SslImpl$UnsecureClientAwareTransportWrapper.process(SslImpl.java:144)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.process(TransportImpl.java:1513)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler$1.run(IOHandler.java:233)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.SelectableImpl.readable(SelectableImpl.java:118)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler.handleQuiesced(IOHandler.java:60)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler.onUnhandled(IOHandler.java:389)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.BaseHandler.onReactorQuiesced(BaseHandler.java:87)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.BaseHandler.handle(BaseHandler.java:206)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.EventImpl.dispatch(EventImpl.java:108)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.dispatch(ReactorImpl.java:324)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.process(ReactorImpl.java:292)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: com.microsoft.azure.eventhubs.impl.MessagingFactory$RunReactor.run(MessagingFactory.java:490)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
Jan 10 23:13:31 beta-bridge eventhub_bridge[23621]: java.lang.Thread.run(Thread.java:748)]
Jan 10 23:15:38 beta-bridge eventhub_bridge[23621]: 23:15:38.557 [pool-3-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 23:17:53 beta-bridge eventhub_bridge[23621]: 23:17:53.854 [pool-3-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 23:20:09 beta-bridge eventhub_bridge[23621]: 23:20:09.153 [pool-3-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 23:22:24 beta-bridge eventhub_bridge[23621]: 23:22:24.321 [pool-3-thread-1] WARN  c.m.a.e.impl.ConnectionHandler - onTransportClosed: hostname[<namespace>.servicebus.windows.net:5671], error[connection aborted]
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: 23:24:59.648 [pool-3-thread-1] WARN  c.m.a.e.impl.MessagingFactory - messagingFactory[MessagingFactory5d58d8], hostName[<namespace>.servicebus.windows.net], error[Unhandled exception while processing events in reactor, report this error.
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: java.lang.NullPointerException: uncorrelated channel: 0
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.EventImpl.dispatch(EventImpl.java:112)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.dispatch(ReactorImpl.java:324)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.process(ReactorImpl.java:292)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: com.microsoft.azure.eventhubs.impl.MessagingFactory$RunReactor.run(MessagingFactory.java:490)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: java.lang.Thread.run(Thread.java:748)Cause: uncorrelated channel: 0
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleBegin(TransportImpl.java:1161)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleBegin(TransportImpl.java:67)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.amqp.transport.Begin.invoke(Begin.java:144)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.handleFrame(TransportImpl.java:1402)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.FrameParser.input(FrameParser.java:418)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.FrameParser.process(FrameParser.java:527)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.SaslImpl$SwitchingSaslTransportWrapper.process(SaslImpl.java:805)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.HandshakeSniffingTransportWrapper.process(HandshakeSniffingTransportWrapper.java:101)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SimpleSslTransportWrapper.unwrapInput(SimpleSslTransportWrapper.java:135)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SimpleSslTransportWrapper.process(SimpleSslTransportWrapper.java:361)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.ssl.SslImpl$UnsecureClientAwareTransportWrapper.process(SslImpl.java:144)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.TransportImpl.process(TransportImpl.java:1513)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler$1.run(IOHandler.java:233)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.SelectableImpl.readable(SelectableImpl.java:118)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler.handleQuiesced(IOHandler.java:60)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.IOHandler.onUnhandled(IOHandler.java:389)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.BaseHandler.onReactorQuiesced(BaseHandler.java:87)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.BaseHandler.handle(BaseHandler.java:206)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.engine.impl.EventImpl.dispatch(EventImpl.java:108)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.dispatch(ReactorImpl.java:324)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: org.apache.qpid.proton.reactor.impl.ReactorImpl.process(ReactorImpl.java:292)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: com.microsoft.azure.eventhubs.impl.MessagingFactory$RunReactor.run(MessagingFactory.java:490)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
Jan 10 23:24:59 beta-bridge eventhub_bridge[23621]: java.lang.Thread.run(Thread.java:748)]
```

**Comment by sjkwak** [comment]
Hi @anthonykirby we believe the issue was fixed in the version 2.0. Can you please try the latest release and let us know if you still see the issue?

**Comment by anthonykirby** [comment]
OK thanks @sjkwak , that's useful to know & I'll upgrade ASAP. 

Until I got round to reporting this, I wasn't aware of the more recent versions - the official-looking documentation at https://docs.microsoft.com/en-us/azure/event-hubs/ refers to old versions.  In fact the Java 'send' docs at https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-java-get-started-send even say "The current version is 1.0.2".  

It'd be great if we could get the docs edited to say something like "look at the github pages for the latest version & latest documentation", so that others don't follow the same obvious route that I did.  What's the best way to go about that?
